src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2894,6 +2894,20 @@
     /**
      * The {@code finally} keyword.
      *
+     * <p>For example:</p>
+     * <pre>
+     * try {} finally { }
```

Comment:
please remove whitespace inside the second curly braces

Reply:
Done!